http.gemspec

```diff
@@ -24,6 +24,7 @@ Gem::Specification.new do |gem|
 
   gem.add_runtime_dependency "http_parser.rb", "~> 0.6.0"
   gem.add_runtime_dependency "http-form_data", "~> 1.0.0"
+  gem.add_runtime_dependency "rack-cache",     "~> 1.2"
```

Comment:
This makes rack a hard dependency of http.rb which I don't think I really like. Perhaps you could make `http/cache` an optional require that pulls in `rack-cache` when loaded?